tailwind.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
         500: "#f64c72",
       },
       blue: {
+        25: "#f3f3fd",
```

Comment:
Great! I have been using `bg-blue-50` together with `bg-opacity-50` until now. I will need to change that in the future. Good change here!